frontend-react/src/global.scss

```diff
@@ -7,17 +7,17 @@
 //@import '~uswds/dist/scss/theme/uswds-theme-utilities';
 
 // override (change some of the defaults loaded above)
-@import "styles/uswds_settings";  // _uswds_settings.scss
-
-@import "uswds";
-
-@import "@trussworks/react-uswds/lib/index.css";
+@forward "styles/uswds_settings";  // _uswds_settings.scss
+@forward 'uswds';
 
+@use 'uswds';
+@import "~@trussworks/react-uswds/lib/uswds.css";
+@import "styles/uswds_custom_theme.scss";
 @import "styles/uswds_overrides"; // _uswds_overrides.scss
 @import "styles/color"; // _color.scss
```

Comment:
I was going to say, we should add the colors to `_color.scss` here, but this file isn't even used? I checked all 3 of the SASS variables and they're not referenced anywhere. Might be worth removing.